nixos/modules/services/networking/bind.nix

```diff
@@ -103,15 +265,15 @@ in
       };
 
       listenOn = mkOption {
-        default = ["any"];
```

Comment:
Please don't mix cosmetic changes into the same PR with actual code changes as this makes the diffs uncessarily long and hard to review.

Reply:
Sorry, i assumed separating them on a commit-level was enough. Should i close this PR in the meantime? (note that i initially set it to WIP)

Reply:
fixed with #95050 , will rebase once #95050 was merged.

Reply:
#95050 was merged, i rebased on the current master, however due to this PR being stuck for a year the diff is now slightly messier than before (due to upstream changes)